CHANGELOG.md

```diff
@@ -189,6 +189,8 @@ This release contains the activation blocks for Berlin across all supported test
 * Distributions and maven artifacts have been moved off of bintray [\#1886](https://github.com/hyperledger/besu/pull/1886)
 * admin_peers json RPC response now includes the remote nodes enode URL
 * add support for keccak mining and a ecip1049_dev network [\#1882](https://github.com/hyperledger/besu/pull/1882)
+* Introduced transitions for IBFT2 block rewards [\#1977](https://github.com/hyperledger/besu/pull/1977) 
```

Comment:
This needs to move to 21.7.0-RC1 section now in the changelog

Reply:
Actually since this is a trivial change, I'm happy to just move this to correct place in the changelog before I merge it.